docs/installing-linux.md

```diff
@@ -33,6 +33,8 @@ Ubuntu and Ubuntu-based distributions, such as Mint or ElementaryOS are unsuppor
 2. Plug the USB Drive into the Chromebook.
 3. Turn on the Chromebook, press `ESC` at the POST screen (when the boot logo appears), and select your USB to boot from. 
 4. Install as you would on any other computer.
+5. (Optional) Install Chultrabook Tools for fan controls, backlight controls, System Information and more
```

Comment:
System Information should be all lowercase.

System Information -> system information